src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -363,7 +363,7 @@ public BigTableWriter.Builder builder(Descriptor descriptor)
     // we always incremented the major version.
     static class BigVersion extends Version
     {
-        public static final String current_version = "nc";
+        public static final String current_version = "oa";
```

Comment:
In the absence of [CASSANDRA-18301](https://issues.apache.org/jira/browse/CASSANDRA-18301), this could be controlled by a testing system property, so we can test behaviour with both the old and the new format:
```suggestion
        public static final String earliest_supported_version = System.getProperty("cassandra.test.sstable_version", "nc");
```
We could set that from dtests, and perhaps even add CI jobs running full test suites with different formats.